packages/_private/angular-demo/src/app.component.css

```diff
@@ -1,21 +1,10 @@
-header {
-  align-items: center;
-  background: var(--syn-color-neutral-50);
-  border-bottom: 1px solid var(--syn-color-neutral-400);
+syn-header {
   box-shadow: var(--syn-shadow-large);
```

Comment:
@eileenwegner I'm not sure but we don't add shadows to the `syn-header`, right?

Reply:
There is no shadow defined, but we have a ticket for shadows: https://github.com/synergy-design-system/design/issues/79

Reply:
In this case we should make this clear in the docs. There is no documentation that I am aware of where we say the navigation is supposed or even forced to be fixed, alas we could also add it directly to the component.

Reply:
For the shown apps we just expected it and for the rail-nav there's no other possibility than having it fixed. But you're absolutetly right,this isn't clearly documented and should defintietely be handled when doing the templates.

Reply:
I have removed the box shadow from all demos in the meantime, should be fine now :)